src/frontends/ir/src/input_model.cpp

```diff
@@ -215,6 +215,7 @@ class InputModel::InputModelIRImpl {
         m_opsets["opset7"] = ngraph::get_opset7();
         m_opsets["opset8"] = ngraph::get_opset8();
         m_opsets["opset9"] = ngraph::get_opset9();
+        m_opsets["opset10"] = ngraph::get_opset10();
```

Comment:
@ilyachur can we reuse ops.hpp ? to avoid manual update of this file each release? E.g. adding new op to ops.hpp leads to adding it to IR frontend

Reply:
Should be addressed by @ilyachur  in the separate PR in case of decision